docs/sample-solutions/windows-server/healthcare-multitier-app.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Windows Server on Azure healthcare front-end application
+description: Proven solution for building a secure, multi-tier web application with Windows Server on Azure that uses scale sets, Application Gateway, and load balancers.
+author: iainfoulds
+ms.date: 06/26/2018
+---
+# Deploy a secure and scalable multi-tier web application on Windows Server for a healthcare solution
+
+This sample solution is applicable to healthcare organizations that have a need to secure multi-tier applications. In this solution, a front-end ASP.NET application securely connects to a protected back-end Microsoft SQL Server cluster.
+
+Example application scenarios include running operating room applications, patient appointments and records keeping, or prescription refills and ordering. Traditionally, organizations had to maintain legacy on-premises applications and services for these scenarios. With a secure way and scalable way to deploy these Windows Server applications in Azure, organizations can modernize their deployments are reduce their on-premises operating costs and management overhead.
+
+## Potential use cases
+
+You should consider this solution for the following use cases:
+
+* Modernizing application deployments in a secure cloud environment.
+* Reducing legacy on-premises application and service management.
+* Improving patient healthcare and experience with new application platforms.
+
+## Architecture diagram
+
+The solution diagram below is an example of this solution:
```

Comment:
Should link somewhere to our [N-tier SQL Server](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/n-tier/n-tier-sql-server) reference architecture

Reply:
Added link. This somewhat begs the question why to have a separate solutions doc for this since that RA is pretty comprehensive and includes a deployment template. 

@david-stanford - thoughts?

Reply:
These are intended to be lighter weight than the RAs and specific.  RAs are very generic overall (n-tier for example).

Reply:
Okay, makes sense. Thanks.